src/main/resources/feedbackQuestionTextEditFormTemplate.html

```diff
@@ -0,0 +1,10 @@
+<div>
+    <br>
+    <div class="row">
+        <div class="col-sm-4" data-toggle="tooltip" data-placement="top" title="recommended length ">Recommended Length:
```

Comment:
The tool tip can indicate that the recommended length is shown to the respondent but not enforced.